dotCMS/src/main/java/com/dotcms/graphql/business/NavigationTypeProvider.java

```diff
@@ -47,15 +47,15 @@ private Map<String, GraphQLOutputType> createNavigationFields() {
 
     final Map<String, GraphQLOutputType> navigationFields = createNavigationFields();
 
-    final GraphQLObjectType navigationType = createObjectType("Navigation", navigationFields, null);
+    final GraphQLObjectType navigationType = createObjectType("DotNavigationAPI", navigationFields, null);
```

Comment:
Sounds like "DotNavigationAPI" can be a constant instead of repeating it three times

Reply:
Done